src/redux/actions/backendStatus/loadBackendStatus.js

```diff
@@ -17,7 +17,9 @@ const loadBackendStatus = (experimentId) => async (dispatch) => {
 
   const url = `/v1/experiments/${experimentId}/backendStatus`;
   try {
-    const response = await fetchAPI(url);
+    const response = await fetchAPI(url, {}, {
+      uiUrl,
```

Comment:
What is this being used for?

Reply:
these are changes introduced by Marcell from this PR -https://github.com/biomage-ltd/ui/pull/499

Reply:
still, do figure out what this is for